core/admin.py

```diff
@@ -119,7 +145,10 @@ def dataset(self, obj):
         return obj.algorithm_job.dataset
 
     def log_preview(self, obj):
-        return _text_preview(obj.log)
+        return _text_preview(obj.log, obj.data_mimetype, True)
```

Comment:
We don't want to pass the data_mimetype for the log, since it only applies to the data.  We can assume logs are supposed to be readable.

This applies to the other class that has a `log_preview` function, too.